src/entrypoints/main/index.html

```diff
@@ -31,6 +31,9 @@
 			  document.write(
 			    '<link rel="icon" type="image/x-icon" href="/img/favicon.ico">'
 			  );
+			  document.write(
+    '\x3Cscript type="text/javascript">questback.popup.create("https://response.questback.com/difi/kirzcfibc7", {title: "Har du 2 minutter?",text: "Vi ønsker tilbakemelding på arbeidet vi gjør for å formidle åpne data fra det offentlige, og hjelp til å forbedre innsatsen. Vil du delta på en kort brukerundersøkelse?", buttons: [{type: "participate",text: "Ja"},{type: "decline",text: "Nei"}]});\x3C/script>'
```

Comment:
\x3C ?

Reply:
Hentet denne fra den tidligere questback-commiten til Øyvind. Fant denne begrunnelsen på nett:
https://stackoverflow.com/questions/8231048/why-use-x3c-instead-of-when-generating-html-from-javascript